charts/sda-mq/templates/statefulset.yaml

```diff
@@ -110,18 +107,18 @@ spec:
               - /bin/bash
               - -ec
               - rabbitmq-diagnostics -q ping
-          initialDelaySeconds: 20
-          periodSeconds: 5
-          timeoutSeconds: 5
+          initialDelaySeconds: 30
```

Comment:
these seem arbitrary, were they a result of some issue with a deployment on a specific system?